tests/UI/pages/BO/orders/view/viewOrderBasePage.ts

```diff
@@ -1,12 +1,61 @@
-require('module-alias/register');
-const BOBasePage = require('@pages/BO/BObasePage');
+import BOBasePage from '@pages/BO/BObasePage';
+
+import type {Page} from 'playwright';
 
 /**
  * View order base page, contains functions that can be used on view/edit order page
  * @class
  * @extends BOBasePage
  */
-class ViewOrderBasePage extends BOBasePage {
+export default class ViewOrderBasePage extends BOBasePage {
```

Comment:
```suggestion
class ViewOrderBasePage extends BOBasePage {
```

to not export it 2 times maybe

Reply:
Fixed